packages/grafana-ui/src/components/Section/Section.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { SectionFill } from './SectionFill';
+import { SectionLabel } from './SectionLabel';
+
+/**
+ * @alpha
+ */
+export const Section = ({
+  label,
+  children,
+  fill,
+}: {
+  label: string;
+  children: React.ReactNode;
+  inline?: boolean;
+  fill?: boolean;
+}) => (
+  <div className="gf-form-inline">
```

Comment:
If possible, use Emotion instead. I think `getFormStyles` might be of some help. The styles might be incompatible, so maybe that will have to be the next iteration.

Reply:
Updated in 9822fed23ebdc760602de8aecedf9b50abdf03e9. It's using `useStyles2`. I couldn't use any of `getFormStyles` styles - shall I create a new entry there?